client/src/app/UI/side-panel/tabs/roi/roiitem/roiitem.component.ts

```diff
@@ -758,8 +758,8 @@ export class ROIItemComponent implements OnInit
         return this.regionLayer.roi.mistROIItem?.ID_Depth || null;
     }
 
-    get dateAdded(): string
+    get dateAdded(): number
     {
-        return this.regionLayer.roi?.dateAdded;
+        return this.regionLayer.roi.createUnixTimeSec*1000;
```

Comment:
I converted `dateAdded` to a `number` type in my hotfix branch that I already merged in since it was related (I believe that's what all of the merge conflicts now are with this branch). Feel free to prefer all your changes over those when merging this in though

Reply:
Thanks for the heads up!